django_scrubber/scrubbers.py

```diff
@@ -5,7 +5,7 @@
 import importlib
 import logging
 
-import faker
+from faker import Faker as LibFaker, Generator
```

Comment:
I know this is mostly a matter of taste, but can we keep the `import faker` and use `faker.Faker` + `faker.Generator` below?
I just dislike import renaming :wink:

Reply:
Sure: your repository, your styleguide/choice.

This was just a matter of chance anyway, didn't decide to import this way consciously, was just experimenting a bit.
I'll change that and the version range tomorrow afternoon.